templates/todo/tag_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+    <h1>{{ object|yesno:"Update,Create" }} Tag</h1>
```

Comment:
https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#yesno please make sure you are aware of `None` behavior.
spoiler: None will be converted to `False` then `Create` will be chosen.